room.h

```diff
@@ -90,6 +90,7 @@ namespace QMatrixClient
             QList<User*> membersLeft() const;
 
             Q_INVOKABLE QList<User*> users() const;
+			Q_INVOKABLE QStringList usernames() const;
```

Comment:
If I try to call `users()`, I get the error "Unknown method return type: QList<User*>" and not sure if Qt even supports such a thing.